install.md

```diff
@@ -123,12 +123,30 @@ helm init --wait --service-account tiller
 
 ## Installing Kritis to your cluster
 
-Install the `resolve-tags` plug-in:
+Install the `resolve-tags` kubectl plugin and binary:
+
+## Mac OS X
+
+```shell
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-darwin-amd64.tar.gz && \
```

Comment:
Not necessary to solve for now, but in the future, we should consider bundling the resolve-tags plugin inside of the kritis release, so that folks don't have to download it twice.